src/monic.jl

```diff
@@ -0,0 +1,38 @@
+struct Monic{T,OPs<:AbstractQuasiMatrix{T},NL} <: OrthogonalPolynomial{T}
+    P::Normalized{T,OPs,NL}
```

Comment:
Does this actually benefit from requiring normalized OPs?

Reply:
Makes the code for getting the recurrence coefficients simpler I think